modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/Sidebar.js

```diff
@@ -92,21 +95,19 @@ export default function Sidebar() {
 	);
 
 	useEffect(() => {
-		const productMenuOpen = document.querySelector('.product-menu-open');
 		const sideNavigation = Liferay.SideNavigation.instance(
 			document.querySelector('.product-menu-toggle')
 		);
 
 		const onCloseSidebar = () => {
 			dispatch(
 				Actions.switchSidebarPanel({
-					sidebarOpen: false,
-					sidebarPanelId: null
+					sidebarOpen: false
 				})
 			);
 		};
 
-		if (productMenuOpen && sidebarOpen) {
+		if (sideNavigation.visible()) {
```

Comment:
Do we need to check if the product menu is open in every render of those components? Wouldn't it better to rely on the sidebarOpen property from the store like before?

What do you think about rendering the sidebar closed as default and open it only if the product menu is not opened? :)